deepchecks/utils/dataframes.py

```diff
@@ -126,3 +126,29 @@ def select_from_dataframe(
         return df.drop(labels=ignore_columns, axis='columns')
     else:
         return df
+
+
+def generalized_corrwith(x1: pd.DataFrame, x2: pd.DataFrame, method: t.Callable):
+    """
+    Compute pairwise correlation.
+
+    Pairwise correlation is computed between columns of one DataFrame with columns of another DataFrame.
+    Pandas' method corrwith only applies when both dataframes have the same column names,
+    this generalized method applies to any two Dataframes with the same number of rows, regardless of the column names.
+
+    Parameters
+    __________
```

Comment:
I think it's in the wrong format. should be ---
Don't know if this is github or for real...

Reply:
It's probably Github, in the real code it looks like this
![image](https://user-images.githubusercontent.com/99395146/173554681-c8114b8b-5907-4e61-86e4-c4dc161d2355.png)